### ISSUE TITLE
Extension isn't working

### ISSUE THREAD

**Issue by d9i** [bencooper222/firefox-address-bar-expression-parser#19]
Just tried this extension. v105.0.3 (64-bit), MacOS. 
Shows address bar prompt but doesn't offer any solutions to expressions

<img width="1455" alt="image" src="https://user-images.githubusercontent.com/1351693/197417603-56e065c7-12e9-432c-ab9f-1aded9758609.png">

Any help would be appreciated!

**Comment by bencooper222** [comment]
Yeah it's blocked on some firefox code signing requirement so they regressed to a version that doesn't work :/ I'll prioritize fixing this sometime this week
